rocblas/PKGBUILD

```diff
@@ -12,8 +12,10 @@ makedepends=('cmake' 'git' 'python' 'python-pip' 'python-virtualenv' 'python-pya
              'perl-file-which' 'msgpack-c' 'rocm-cmake' 'gcc-fortran')
 _rocblas='https://github.com/ROCmSoftwarePlatform/rocBLAS'
 source=("$pkgname-$pkgver.tar.gz::$_rocblas/archive/rocm-$pkgver.tar.gz"
-        "include-path.patch::https://github.com/ROCmSoftwarePlatform/rocBLAS/commit/992429ff1d04195b10f9a3350668e180b34dbdb5.patch")
+        "include-path.patch::https://github.com/ROCmSoftwarePlatform/rocBLAS/commit/992429ff1d04195b10f9a3350668e180b34dbdb5.patch"
+        "gfx1031.patch::https://github.com/littlewu2508/rocBLAS/commit/6c360dca36e8d6e691527b238d9a22f656ff3ade.patch")
```

Comment:
Can you pull from the PR (https://patch-diff.githubusercontent.com/raw/ROCmSoftwarePlatform/rocBLAS/pull/1251.patch), instead of the local commit directly from @littlewu2508 's repo?